### PR TITLE
Reduce yaml files for lvm+RAID1

### PR DESCRIPTION
**Pull request by okynos**
We have unified all lvm+RAID1 YAMLs files into single one.

- Related ticket: https://progress.opensuse.org/issues/163250
- MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/306
- Needles: N/A
- Verification run:
  - Flavor Online -> https://openqa.suse.de/tests/overview?distri=sle&version=15-SP6&build=93.5&groupid=583
    - lvm+RAID1:
      - x86_64: https://openqa.suse.de/tests/15167377 (Updated-13/08/2024)
      - aarch64: https://openqa.suse.de/tests/15167376 (Updated-13/08/2024)
    - lvm+RAID1@svirt-xen-hvm: https://openqa.suse.de/tests/15167371 (Updated-13/08/2024)
    - lvm+RAID1@svirt-xen-pv: https://openqa.suse.de/tests/15167086 (Updated-13/08/2024)
    - lvm+RAID1@uefi: https://openqa.suse.de/tests/15167085 (Updated-13/08/2024)

### CODE REVIEW

File: schedule/yam/test_cases/lvm+raid1_sle15.yaml [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
   Validation of partitioning for raid1 on lvm
   Installation of RAID1 using expert partitioner.
-vars:
-  YUI_REST_API: 1
 schedule:
   suggested_partitioning:
     - installation/partitioning/setup_raid1_lvm
+  security:
+  - installation/installation_settings/validate_ssh_service_enabled
+  - installation/installation_settings/open_ssh_port
```
Comment (jknphy) [inline schedule/yam/test_cases/lvm+raid1_sle15.yaml:11]: these two are not common to old, are default for some particular default schedule.

Comment (okynos) [reply schedule/yam/test_cases/lvm+raid1_sle15.yaml:11]: As I see the `default_svirt-hyperv.yaml` contains both steps, but the related jobs don't mention any default schedule (they are inside `SLE_15_DEV`) If `SLE_15_DEV` is only for development and not for production we could safely remove both steps from the file. What do you think?

Comment (jknphy) [reply schedule/yam/test_cases/lvm+raid1_sle15.yaml:11]: yep, we gave up in some point with the hypervisor and reported to that squad, we used to have more jobs with that. I agree then, thx.

Comment (okynos) [reply schedule/yam/test_cases/lvm+raid1_sle15.yaml]: Solved here 20562c20e4b65aedf39a85141813b548daaf03ab

Thanks

**Review by jknphy — APPROVED** [review]

**Pull request merged**